jobs/payment-jobs/utils/enums.py

```diff
@@ -0,0 +1,21 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Enum definitions."""
+from enum import Enum
+
+
+class MailerEvents(Enum):
+    """Mailer event types."""
+
+    EFT_INVOICE_CREATED = 'eft.invoiceCreated'
```

Comment:
This exists in pay-api

Reply:
https://github.com/bcgov/sbc-pay/blob/2c9fcd89ab8498defd0660d7caf509b6f8b8706d/pay-api/src/pay_api/utils/enums.py#L341